gossip/src/crds_gossip_pull.rs

```diff
@@ -1108,8 +1109,9 @@ pub(crate) mod tests {
         let now: u64 = 1_605_127_770_789;
         let thread_pool = ThreadPoolBuilder::new().build().unwrap();
         let mut ping_cache = PingCache::new(
-            Duration::from_secs(20 * 60), // ttl
-            128,                          // capacity
+            Duration::from_secs(20 * 60),      // ttl
+            Duration::from_secs(20 * 60) / 64, // ttl
```

Comment:
both lines are commented as `ttl`.
also would be much better to use `rate_limit_delay` instead of `delay`, so that it would be more clear and easier to find all references across files.

Reply:
missed that one, updated to `rate_limit_delay`